docker/dev/nodejs/Dockerfile

```diff
@@ -11,6 +11,7 @@ ADD ./karma.conf.js /code
 
 # Install Prerequisites
 RUN npm install -g bower gulp
+RUN apt-get install gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
```

Comment:
This command is causing docker build to fail. 
<img width="1919" alt="Screenshot 2019-05-14 at 7 26 02 PM" src="https://user-images.githubusercontent.com/11664259/57740572-262dbf80-767e-11e9-9a8f-259b358e0fdb.png">